src/libraries/Common/tests/System/Security/Cryptography/AlgorithmImplementations/Symmetric/SymmetricOneShotBase.cs

```diff
@@ -0,0 +1,383 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+using System.Security.Cryptography;
+using Test.Cryptography;
+using Xunit;
+
+namespace System.Security.Cryptography.Tests
+{
+    public abstract class SymmetricOneShotBase
+    {
+        protected abstract byte[] Key { get; }
+        protected abstract byte[] Iv { get; }
```

Comment:
Let's match existing casing :smile:

```suggestion
        protected abstract byte[] IV { get; }
```

It's also a two letter abbreviation that isn't "Id", so UU is the guideline casing anyways.